contracts/SmartSession.sol

```diff
@@ -91,7 +92,7 @@ contract SmartSession is SmartSessionBase {
         // If the SmartSession.USE mode was selected, no futher policies have to be enabled.
         // We can go straight to userOp validation
         // This condition is the average case, so should be handled as the first condition
-        if (mode == SmartSessionMode.USE) {
+        if (mode.isUseMode()) {
```

Comment:
not sure if this is actually better tbh

Reply:
this one may be not, yes. I wanted everything to be uniform 
since I'm using library functions next, decided to use a function here as well